### ISSUE TITLE
Decoding issue with get_email

### ISSUE THREAD

**Issue by willstott101** [django-helpdesk/django-helpdesk#508]
The code here:
https://github.com/django-helpdesk/django-helpdesk/blame/master/helpdesk/management/commands/get_email.py#L324

Seems to be expecting to receive escaped unicode. However in my environment that doesn't appear to be case. The committed code raises a UnicodeEncodeError when reading unicode emails.

My workaround:

```
                if six.PY3:
                    try:
                        # Original workaround to get unicode text out rather than escaped text
                        body = body.encode('ascii').decode('unicode_escape')
                    except UnicodeEncodeError:
                        pass # >>> Support receiving real unicode too...
                else:
                    body = body.encode('utf-8')
```

Python 3
Django (1.8.12)
email-reply-parser (0.5.9)

**Comment by gwasser** [comment]
Hi @willstott101 thanks for the ticket. Do you happen to have an example minimal email or environment settings that illustrate the bug? We're trying to expand our testing so we can fix up all of these little bugs and workarounds.

**Comment by gwasser** [comment]
I believe this was addressed by a similar workaround in #529 . Because of that I will close this ticket for now, but please re-open if the problem still exists. If you do run into trouble please post a sample email that recreates the bug so we can add it to our unit tests and catch it in the future.
